### DISCUSSION TITLE
Back to top button

### DISCUSSION THREAD

**Discussion by norecords** [poblabs/weewx-belchertown#845] [Show and tell]
Good news everyone!
I'll publish a pull request asap with my work.
see it [Live](https://meteo.correns.org).

**Comment by 41south** [comment]
Very nice, where is it available?

**Reply by norecords** [reply]
I've changed JavaScript to jQuery, also added opacity to `#btn-back-to-top` css.
I'm preparing a PR with option in skin.conf to display it and let choose left or right side.
See that PR https://github.com/poblabs/weewx-belchertown/pull/846

**Comment by norecords** [comment]
This have been merged in realese 1.3
